### ISSUE TITLE
Erection at time of symptom onset

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10011]
**Preferred term label:**
Erection at time of symptom onset

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Penis fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this not rather rapid loss of erection?
see https://www.mayoclinic.org/healthy-lifestyle/sexual-health/expert-answers/penis-fracture/faq-20058154

**Comment by MickeySegal** [comment]
The point of this finding is to describe the fact that the injury can only occur when the penis is erect.  As your citation said "a penis fracture can occur when there is trauma to an **erect** penis".  It is a "what were you doing when this happened" finding.

**Comment by pnrobinson** [comment]
@MickeySegal 
It seems to me that this
Immediate detumescence of penis at time of symptom onset #10013
is sufficient, and if this code is used than it already implies Erection at symptom onset?

**Comment by MickeySegal** [comment]
OK.  this is the only place we use this finding so I will delete.

**Comment by pnrobinson** [comment]
closing
